src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/Target_X64/X64Emitter.cs

```diff
@@ -98,6 +98,38 @@ public void EmitJmpToAddrMode(ref AddrMode addrMode)
             EmitIndirInstruction(0xFF, 0x4, ref addrMode);
         }
 
+        public void EmitPUSH(sbyte imm8)
+        {
+            Builder.EmitByte(0x6A);
+            Builder.EmitByte(unchecked((byte)imm8));
+        }
+
+        public void EmitPUSH(ISymbolNode node)
```

Comment:
Using the existing `EmitIndirInstruction`/`EmitLEAQ` where possible would result in fewer `EmitByte`s here, but that's a nice to have.